app/views/schools/index.html.erb

```diff
@@ -11,7 +11,7 @@
 <div class="col-md-12">
 
   <p><%= t('schools.index.introduction_1') %>.</p>
-  <p><%= t('schools.index.introduction_2', count: @schools.count) %>.</p>
+  <p><%= t('schools.index.introduction_2', count: School.visible.count) %>.</p>
```

Comment:
Would @schools.length be more efficient here as you already have @schools rather than doing another query? I might be wrong... I suppose a count would be quite minimal

Reply:
Yes, that would be better.